python/BioSimSpace/FreeEnergy/_relative.py

```diff
@@ -795,23 +1523,23 @@ def _initialise_runner(self, system):
                 # Create a copy of the process and update the working
                 # directory.
                 if not self._setup_only:
-                    process                 = _copy.copy(first_process)
-                    process._system         = first_process._system.copy()
-                    process._protocol       = self._protocol
-                    process._work_dir       = new_dir
-                    process._std_out_file   = new_dir + "/somd.out"
-                    process._std_err_file   = new_dir + "/somd.err"
-                    process._rst_file       = new_dir + "/somd.rst7"
-                    process._top_file       = new_dir + "/somd.prm7"
-                    process._traj_file      = new_dir + "/traj000000001.dcd"
-                    process._restart_file   = new_dir + "/latest.rst"
-                    process._config_file    = new_dir + "/somd.cfg"
-                    process._pert_file      = new_dir + "/somd.pert"
+                    process = _copy.copy(first_process)
+                    process._system = first_process._system.copy()
+                    process._protocol = self._protocol
+                    process._work_dir = new_dir
+                    process._std_out_file = new_dir + "/somd.out"
+                    process._std_err_file = new_dir + "/somd.err"
+                    process._rst_file = new_dir + "/somd.rst7"
+                    process._top_file = new_dir + "/somd.prm7"
+                    process._traj_file = new_dir + "/traj000000001.dcd"
+                    process._restart_file = new_dir + "/latest.rst"
+                    process._config_file = new_dir + "/somd.cfg"
+                    process._pert_file = new_dir + "/somd.pert"
```

Comment:
I'm not sure if this was the result of a whitespace trim? The previous alignment was intended to aid readability

Reply:
Ah yes that must have been the automatic formatting, I've changed it back.